dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -657,7 +672,10 @@ internal open class RumViewScope(
             dd = ViewEvent.Dd(
                 documentVersion = version,
                 session = ViewEvent.DdSession(plan = ViewEvent.Plan.PLAN_1)
-            )
+            ),
+            connectivity = networkInfo.toViewConnectivity(),
```

Comment:
I had considered it would be useful to get the last connectivity value the view had instead of no information, but like you said it's probably not all that actionable and most insight can already be taken from resource connectivity values. I'll remove it.